chain/find.go

```diff
@@ -45,40 +45,45 @@ func NewGapIndexer(node lens.API, db *storage.Database, name string, minHeight,
 	}
 }
 
-func (g *GapIndexer) Run(ctx context.Context) error {
-	startTime := time.Now()
-
-	head, err := g.node.ChainHead(ctx)
-	if err != nil {
-		return err
-	}
-	if uint64(head.Height()) < g.maxHeight {
-		return xerrors.Errorf("cannot look for gaps beyond chain head height %d", head.Height())
-	}
-
+func (g *GapIndexer) findGapsAndNullRounds(ctx context.Context) (visor.GapReportList, []abi.ChainEpoch, error) {
```

Comment:
I don't think you meant to change this from `Run` to `findGapsAndNullRounds` did you?

Reply:
Ugh, I see it now. Sry, thx.